demo/src/main/java/foundation/algorand/demo/AnswerActivity.kt

```diff
@@ -75,6 +75,14 @@ class AnswerActivity : AppCompatActivity() {
 
     private val cookieJar = Cookies()
 
+    private val iceServers = listOf(
+        PeerConnection.IceServer.builder("stun:stun.relay.metered.ca:80").createIceServer(),
```

Comment:
Same as the JS, didn't realize it had a STUN server. Let's keep the google `stun` servers and remove the `metered` stun service.

Reply:
You got it amigo